pulsar-broker/src/test/java/org/apache/pulsar/client/impl/TableViewTest.java

```diff
@@ -86,7 +90,7 @@ protected void cleanup() throws Exception {
         try (Producer<byte[]> producer = builder.create()) {
             CompletableFuture<?> lastFuture = null;
             for (int i = 0; i < count; i++) {
-                String key = "key"+ i;
+                String key = "key" + i;
```

Comment:
Please undo the unrelated change.

Reply:
Uh, my autoformatter somehow kicked in again, thanks for catching!